src/content/support/p1-process/index.md

```diff
@@ -0,0 +1,92 @@
+
+## Separation of Responsibilities
+
+It’s important to make sure that everybody involved in an incident knows their role, what is expected from them and that it doesn’t conflict with others' responsibilities. Somewhat counterintuitively, a clear separation of responsibilities allows individuals more autonomy, as they do not need to ask all the time and coordinate actions.
+
+### Roles
+
+At Giant Swarm we have two defined roles: Incident Coordinators and Operations Engineers.
+
+### Incident Coordinator
+
+The Incident Coordinator (IC) holds the high-level state about the incident. They structure the incident response, assigning responsibilities according to need and priority. De facto, the IC holds all positions/responsibilities that they have not delegated. If appropriate, they can remove roadblocks that prevent Operations Engineers from working most effectively.
+
+The IC is the public face of our incident response. It is within the ICs duty to issue periodic updates to the team involved - both customer teams and within Giant Swarm - to act as the bridge between the customer and the team. The IC will need to be in the war rooms of our customers.
+
+If there is a dedicated IC, the IC isn’t debugging systems and keeps the focus on coordinating the team to do so, while managing our customers.
+
+### Operations Engineer
+
+The Operations Engineer (OE) works with the incident coordinator to respond to the incident and is the only one debugging and applying changes to a system.
+
+## P1 Incident Process
+
+The process starts from the well-known [Incident Command System](https://en.wikipedia.org/wiki/Incident_Command_System) used by US firefighters to manage emergency situations. Obviously we have adapted to manage developer platforms. 
+
+Our main tenet is to have a simple process integrated with our incident tooling ([Incident.io](https://incident.io/)) to simplify the life of our engineers. Once a critical incident is declared none wants to read a process but be driving by it. 
+
+The process can be broken down in these steps:
+
+1. [Identify](#identify)
+2. [Investigate](#investigate)
+3. [Fixing](#fixing)
+4. [Monitoring](#monitoring)
+5. [Closing up](#closing-up)
+
+### Identify
+
+First step is to identify the incident and understand its impact and severity. In our case, there are three possible sources:
+
+- Alert received pointing to an impacted production system
+- Customer sent us an urgent email
+- Customer reached out via Slack
+
+Once our engineer acknowledge the message we need to confirm the severity to trigger the P1 process. In case the customer paged us, the Giant Swarm engineer joins the call or thread to verify the severity. In case an alert paged us, the engineer connects to the platform and measure the impact.
```

Comment:
```suggestion
Once our engineer acknowledges the message we need to confirm the severity to trigger the P1 process. In case the customer paged us, the Giant Swarm engineer joins the call or thread to verify the severity. In case an alert paged us, the engineer connects to the platform and assesses the impact.
```